tags/fun/dfu.ytag

```diff
@@ -0,0 +1,7 @@
+type: text
+
+---
+
+<:lambdasoup:798677550159102012><:steve_lambda:691095252458537011><:lambdasoup2:798683897730564136>
+
+https://youtu.be/lClqn2k6W_4
```

Comment:
no use https://youtu.be/ON1vN9NnCc8

Reply:
```suggestion
https://youtu.be/ON1vN9NnCc8
```
Just putting this here so I can add it quickly